include/rocksdb/listener.h

```diff
@@ -86,6 +86,23 @@ enum class BackgroundErrorReason {
   kMemTable,
 };
 
+enum class WriteStallCondition {
+  kNormal,
+  kDelayed,
+  kCompactionPressure,
```

Comment:
can remove this - we don't need to notify for compaction pressure. it's only for an internal mechanism to adjust number of compaction threads.